src/lib/autoinstall/autosetup_helpers.rb

```diff
@@ -122,12 +114,12 @@ def semi_auto?(name)
 
     # Autosetup the network
     def autosetup_network
-      @network_before_proposal = false
+      # Prevent to be called twice in case of already configured
+      return if @network_configured
```

Comment:
Maybe I should move the check outside of the method to make it more clear...

https://github.com/yast/yast-autoinstallation/blob/master/src/clients/inst_autosetup.rb#L168

The `autosetup_network` can be actually called in two places. The original call, that is in the autosetup client and in autoinit. So, basically what I wanted to prevent, was calling it again in case of already configured during autotinit.

It could be done in other ways, but I thought it was simplest enough.